### DISCUSSION TITLE
Web context url override

### DISCUSSION THREAD

**Discussion by yroffin** [fosslight/fosslight#299] [Q&A] [answered]
hi,

i've changed web context with server.servlet.contextPath ... it work ... but static files are alway on /
any body know ow to change this ?

**Comment by soimkim** [comment]
@yroffin , Would you tell me which static files' path is always /?

**Comment by soimkim** [comment, marked as answer]
@yroffin , [PR ](https://github.com/fosslight/fosslight/pull/382) with improved [commits related to context path](https://github.com/fosslight/fosslight/pull/382/commits/84a860f08af073e8d99bcbf9f1a18e0e8388b2ce) have been merged. (FOSSLight v1.2.25)
